tests/test_model_functions.py

```diff
@@ -116,12 +120,16 @@ def test_get_state_variables(self, basic_model):
         """ Tests Model.get_state_variables() works on a simple model. """
 
         states = basic_model.get_state_variables()
+        unsorted_states = basic_model.get_state_variables(sort=False)
+        assert set(states) == set(unsorted_states)
```

Comment:
```suggestion
        assert states == unsorted_states
```
Since there's just one!